packages/manager/apps/pci-gateway/src/api/data/gateways.ts

```diff
@@ -22,7 +31,7 @@ export const createGateway = async (
     return data;
   } catch (e) {
     const error = e as AxiosError;
-    throw new Error((error.response.data as { message: string })?.message);
+    throw error.response.data;
```

Comment:
Maybe you should check with axios.isAxiosError(e) and verify if you had a response ?

Reply:
@lionel95200x 
How could you guarantee that `data` contains `class` property, if no type guard does the check?

Reply:
https://github.com/axios/axios?tab=readme-ov-file#typescript